doc/operation-and-maintenance/Cluster-configuration-and-node-management.md

```diff
@@ -53,71 +53,51 @@ Make sure that the range you set provides enough ports for all the nodes in the
 Remember to keep an epmd port open (port 4369) if any firewall restrictions are required.
 Epmd keeps track of which erlang node is using which ports on the local machine.
 
-## Load Balancing
-
-### Elastic Load Balancer (ELB)
-
-When using ELB pleas be advised that many of our users reported it to prematurely drop connections, especially when using BOSH.
-
-### DNS-based load balancing
-
-Load balancing can be performed on a DNS level. 
-A DNS response can have a number of IP addresses that can be returned to the client side in a random order. 
-
-On the AWS stack this type of balancing is provided by Route53. 
-The description of their service can be found at:
-
-http://docs.aws.amazon.com/Route53/latest/DeveloperGuide/WeightedResourceRecordSets.html
-
-### Software load balancer
-
-A good example of load balancing on the application layer are HAProxy and Nginx. 
-
-### Other
-
-The approaches described above can be mixed - we can use DNS load balancing to pick a software load balancer which will select one of the nodes.
-
 ## Connecting nodes
 
 Checklist:
 
-- working directory `rel/mongooseim`
+- working directory `rel/mongooseim` (root of MongooseIM release or installation)
```

Comment:
working directory `rel/mongooseim` (root of a MongooseIM release or installation)